src/common/image.c

```diff
@@ -971,6 +976,31 @@ void dt_image_init(dt_image_t *img)
   img->cache_entry = 0;
 }
 
+void dt_image_refresh_makermodel(dt_image_t *img)
+{
+  char *maker = NULL; 
+  char *model = NULL;
+
+  if (img->raw_maker[0] != 0 && img->raw_model[0] != 0)
```

Comment:
It's probably the same thing on most platforms but the code is actually correct because those values are initialized with memset(0).